src/Models/Scopes/Product/WithProductSuperAttributesScope.php

```diff
@@ -53,6 +53,7 @@ public function apply(Builder $builder, Model $model)
                     ->where('catalog_product_super_attribute_label.store_id', config('rapidez.store'));
             })
             ->whereColumn('product_id', $model->getTable() . '.entity_id')
+            ->orderBy('frontend_label')
```

Comment:
This extra orderBy makes sure the sorting is the same as the standard magento functionality: If `sort_order` is the same everywhere, it gets sorted alphabetically.